skrub/_reporting/_data/templates/column-summaries.css

```diff
@@ -4,9 +4,18 @@
 /* Collection of column summary cards (as shown in the "column summaries" tab) */
 /* --------------------------------------------------------------------------- */
 
+/* Used as a container to enable horizontal scroll of the column
+ * summaries*/
+.horizontal-scroll {
```

Comment:
there is already a horizontal-scroll class in base.css that makes an element scrollable horizontally. the padding at the bottom is not really related to that so should maybe be applied separately

Reply:
Fair point, I hadn't seen it (my bad, I should have "git grep"ed better).

I've refactor for a cleaner solution